docs/development/contributor_guidelines.rst

```diff
@@ -29,6 +29,45 @@ your PR. If you modify the server configuration, you should run the
 testinfra tests. Please denote in the checklist when you submit the PR that
 you have performed these checks locally.
 
+Code Style
+----------
+
+We use code linters to keep a consistent code quality and style. These linters
+also run in CI and will produce build failures. To avoid this, we suggest running
+the linters locally as you work.
+
+.. note::
+  On the development VM, the code linters we use are installed. To
+  install the code linters locally on your host machine, from the root of the
+  repo you can run the following:
+
+  .. code:: sh
+
+     pip install securedrop/requirements/develop-requirements.txt
```

Comment:
The note text is a bit confusing, as well. I suggest this patch:

```
diff --git a/docs/development/contributor_guidelines.rst b/docs/development/contributor_guidelines.rst
index bd841cd6..104d2790 100644
--- a/docs/development/contributor_guidelines.rst
+++ b/docs/development/contributor_guidelines.rst
@@ -37,13 +37,13 @@ also run in CI and will produce build failures. To avoid this, we suggest runnin
 the linters locally as you work.
 
 .. note::
-  On the development VM, the code linters we use are installed. To
-  install the code linters locally on your host machine, from the root of the
-  repo you can run the following:
+  The code linters are installed automatically on the Development VM.
+  To install the linting tools locally on your host machine, from the root
+  of the repo you can run the following:
 
   .. code:: sh
 
-     pip install securedrop/requirements/develop-requirements.txt
+     pip -r install securedrop/requirements/develop-requirements.txt
 
 Python
 ~~~~~~
```